.travis/download_vst_sdk.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -ex
+
+vst_sdk_archive="vst-sdk_3.6.14_build-24_2019-11-29.zip"
```

Comment:
This archive is expected to be kept in travis cache. Skip download if it exists.

Reply:
Done :)